.env.n8n.example

```
@@ -0,0 +1,41 @@
+N8N_BASIC_AUTH_ACTIVE=true
+N8N_BASIC_AUTH_USER=forge-admin
+N8N_BASIC_AUTH_PASSWORD=CHANGE_ME_USE_STRONG_PASSWORD
+
+N8N_HOST=localhost
+N8N_PORT=5678
+N8N_PROTOCOL=http
+N8N_SECURE_COOKIE=false
+
+# Execution data retention (days)
+EXECUTIONS_DATA_PRUNE=true
+EXECUTIONS_DATA_MAX_AGE=168
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXECUTIONS_DATA_MAX_AGE key should go before the EXECUTIONS_DATA_PRUNE key

(UnorderedKey)